app/views/includes/deprecation_message.html

```diff
@@ -1,5 +1,14 @@
 <div class="notification-summary">
 <h2 class="heading-medium no-top-margin">GOV.UK Elements has now been replaced by the GOV.UK Design System</h2>
-<p>Elements will remain available in case you are currently using it, but is no longer maintained. The Government Digital Service will only carry out major bug fixes and security patches.</p>
-<p>The <a href="https://www.gov.uk/design-system">GOV.UK Design System</a> will be updated to ensure the things it contains meet level AA of WCAG 2.1, but Elements will not. <a href="https://design-system.service.gov.uk/accessibility/">Read more about accessibility of the GOV.UK Design System</a>.</p>
+
+<p>GOV.UK Elements is:<p>
+
+<ul>
```

Comment:
This will need the [bullet list style ](<ul class="list list-bullet">
 ) applied. Once that's added, the list will have 45px bottom margin (from `.list-bullet`) which makes it look a bit unbalanced.

![Screenshot 2020-04-22 at 15 39 33](https://user-images.githubusercontent.com/5007934/79995780-8abce580-84af-11ea-8d73-a28ef950d92d.png)

If we want to fix that, we'll need to add a new class [Elements documentation Sass](https://github.com/alphagov/govuk_elements/blob/master/assets/sass/elements-documentation.scss) as Elements doesn't have spacing override classes. I've just gone through the Elements sass files but I can't see another class we could use here to remove the bottom margin.

Reply:
Thanks for spotting - I've added the CSS classes - let me know if I've done it correctly. Are you happy to leave the margin as is? Feels like it'd probably be going too far to update Elements just for this margin.

Reply:
I think I'd agree – or, if we think it's important, can we just add an inline style to the `ul`?

Reply:
@m-green Happy for you to leave it as is, or like @36degrees suggested you can add `style="margin-bottom: 20px;"` to the `ul` element.